charts/victoria-logs-single/templates/_helpers.tpl

```diff
@@ -3,6 +3,14 @@
   {{- $app := $Values.server -}}
   {{- $args := default dict -}}
   {{- $_ := set $args "retentionPeriod" $app.retentionPeriod -}}
+  {{- with $app.retentionDiskUsage -}}
```

Comment:
people can have the retention in days/months AND in disk space together but must have at least once